### ISSUE TITLE
[jss-cli] Output a standalone "jss sheet" module?

### ISSUE THREAD

**Issue by amio** [cssinjs/jss#280]
I don't know if I can generate a standalone "jss sheet" module, which doesn't depend on the `jss` module, so I can bundle it within my component for a minimal filesize.

In other words, it would be a cjs/es6 module(currently `jss-cli` output format) plus some basic api (`.classes`, `.attach()`, `.detach()`), so I imagine using it like so:

```bash
# compile the styles into a standalone "jss sheet" module
$ jss convert component-styles.css -f standlone > component-styles.js
```

then in the `component.js`:
```jsx
import React from "react"
// here I don't need the `jss` package anymore
import "sheet" from "./component-styles.js"

export default class extends React.Component {
  componentWillMount () {
    sheet.attach()
  }
  componentWillUnmount () {
    sheet.detach()
  }
  render () {
    return <div className={sheet.classes.wrapper}>...</div>
  }
}
```

Or is there already a way to achieve this?

**Comment by kof** [comment]
So you basically want to inline jss library with your component?

**Comment by amio** [comment]
Yes! Only inline these codes used in runtime, without compiler things.

**Comment by kof** [comment]
JSS library is needed for rendering, I don't recommend inlining it into every component, this is a huge overhead in download size. If you want something minimal to render styles identical to css, you may want to use [jss-lite](https://github.com/tomekwi/jss-lite)

**Comment by amio** [comment]
Thanks @kof 

PS, an explanation to my use case:
I'm rewriting a small lib which is just a single component([Console.js](https://github.com/amio/console.js)), and require just one stylesheet. In this case I think the runtime code would be much smaller than full `jss` package to be bundled.
